jenkins/Jenkinsfile

```diff
@@ -137,5 +141,29 @@ pipeline {
         }
       }
     }
+    stage('Rlease Cloudify Cluster Manager'){
+      when { branch pattern: ".+-build", comparator: "REGEXP" }
+      steps{
+        buildClusterManagerRPM("${env.WORKSPACE}", "${env.VERSION}", "ga", "${env.BRANCH}")
+      }
+      post {
+        success {
+          echo 'Upload artifacts to S3'
+          uploadToReleaseS3("${env.WORKSPACE}/rpm/","${env.S3_RELEASE_PATH}")
```

Comment:
@YoniItzhak on which s3 bucket && account this is going to be uploaded ?

Reply:
so as I can see it will be on cloudify-cluster-manager/version-of-cluster-manager-tool/ga-release which means it is on separate folder ? .

Reply:
Right, since this project is separate from our releases, I thought it would be better if it would be on a different folder with its own releases.